framework/framework.go

```diff
@@ -98,36 +98,6 @@ func (f *Framework) ProcessCreate(obj interface{}, resources []Resource) error {
 	return nil
 }
 
-// ProcessCreateWithBackoff is the same as ProcessCreate but takes an additional
-// backoff factory returning new backoff implementations to retry the creation
-// on resource errors. Retries are implemented using a retry resource that wraps
-// each given resource. Errors are logged eventually in case an appropriate
-// logger is configured in the operator framework.
-func (f *Framework) ProcessCreateWithBackoff(obj interface{}, resources []Resource, backoffFactory func() backoff.BackOff) error {
```

Comment:
We recently introduced the resource wrapping. This concept is very powerful and dynamic. Like server middlewares the user is free to implement resource processing to his needs. I thought about the interface here and wanted to get this more straight. Here we remove the specific function we added for retries. Background is further composability. I thought about adding a metrics resource and wondered how to configure this in the framework. It was not possible this way. What makes more sense is to just have the usual Process* methods we had before and provide a way to only wrap resources as needed. I would like to add a metrics resource and wrapping functionality for this in the next PRs.